packages/slice-machine/src/features/onboarding/OnboardingGuide/OnboardingGuide.tsx

```diff
@@ -60,36 +119,51 @@ const confettiConfig: ConfettiConfig = {
   duration: 3000,
 };
 
-const OnboardingGuideContent = () => {
-  const [isVisible, setVisible] = useState(true);
-  const confettiCannonRef = useRef<HTMLDivElement>(null);
-
-  const onComplete = () => {
-    const { current: confettiCannon } = confettiCannonRef;
-    if (confettiCannon) fireConfetti(confettiCannon, confettiConfig);
-    setTimeout(() => setVisible(false), confettiConfig.duration);
-  };
-
-  if (!isVisible) return null;
-
-  return (
-    <OnboardingProvider onComplete={onComplete}>
-      <div className={styles.container}>
-        <OnboardingGuideCard />
-        <div ref={confettiCannonRef} className={styles.confettiCannon} />
-      </div>
-    </OnboardingProvider>
-  );
-};
-
-export const OnboardingGuide = () => {
-  const { eligible } = useOnboardingExperiment();
-  const { sliceMachineUpdateAvailable, adapterUpdateAvailable } =
-    useUpdateAvailable();
+interface CardProps {
+  cardColor?: "grey2";
+  cardVariant?: "outlined" | "animated-light" | "animated-dark";
+  cardTitle: string;
+  cardDescription: string;
+  titleColor?: "grey12" | "purple1";
+  descriptionColor?: "grey11" | "purple6";
+  descriptionVariant?: "small";
+  progressColor?: "white";
+  progressBackgroundColor?: "purple8";
+  progressLabelColor?: "purple3";
+}
 
-  if (!eligible || sliceMachineUpdateAvailable || adapterUpdateAvailable) {
-    return null;
+const getCardPropsFromExperimentVariant = (variant?: string): CardProps => {
+  switch (variant) {
+    case "light":
+      return {
+        cardDescription:
+          "Render a live page with content coming from Prismic in 5 mins",
+        cardTitle: "Build your first Prismic Page in {number} simple steps",
```

Comment:
💡 i prefer the solution i shared above but another option could be to make this a function that receive the steps instead of a string literal, then in the caller side you execute the function to get the value rather than use the value directly